app/Http/Auth/Controllers/LoginController.php

```diff
@@ -46,6 +47,8 @@ protected function sendLoginResponse(Request $request)
 
         flash()->success('You are now logged in');
 
+        app(GrantRayTrialLicenseAction::class)->execute(auth()->user());
```

Comment:
Would this be better in a middleware? It'll be easier to remove just the middleware (or reuse it) later than handling them in the login/register controllers.

Reply:
Going to leave this as it is for now to keep things simple.